src/pages/[lang]/perfezionamento.tsx

```diff
@@ -1,43 +1,176 @@
-import { Infoblock } from "@pagopa/mui-italia";
+import { Infoblock, InfoblockProps } from "@pagopa/mui-italia";
 import { GetStaticPaths, NextPage } from "next";
 import { useRef, useState } from "react";
-import { Box, Fade, Stack } from "@mui/material";
+import { Box, Fade, Link, Stack, Typography } from "@mui/material";
 
-import {
-  getCommonHeadingTitleData,
-  getCommonInfoblockData,
-  getCommonTabsData,
-} from "../../api";
 import HeadingTitle from "../../components/HeadingTitle";
 import Tabs from "../../components/Tabs";
 import PageHead from "../../components/PageHead";
-import { langCodes } from "@utils/constants";
+import { IMAGES_PATH, langCodes } from "@utils/constants";
 import { getI18n } from "../../api/i18n";
-import { LangCode } from "../../model";
+import { IHeadingTitleProps, ITabsProps, LangCode } from "../../model";
+import { useTranslation } from "../../hook/useTranslation";
+import CustomInfoblockContent from "../../components/Perfezionamento/CustomInfoblockContent";
+import CustomInfoblockList from "../../components/Perfezionamento/CustomInfoblockList";
+import CustomInfoblockListItem from "../../components/Perfezionamento/CustomInfoblockListItem";
 
 export const getStaticPaths: GetStaticPaths = async () => {
   return {
     paths: langCodes.map((lang) => ({
       params: { lang },
     })),
     fallback: false,
-  }
-}
+  };
+};
 
-export async function getStaticProps({params}: {params: {lang: LangCode}}) {
-  const translations = getI18n(params.lang, ['common'])
+export async function getStaticProps({
+  params,
+}: {
+  params: { lang: LangCode };
+}) {
+  const translations = getI18n(params.lang, ["common", "perfezionamento"]);
 
-  return { props: {translations, lang: params.lang} }
+  return { props: { translations, lang: params.lang } };
 }
 
 const Perfezionamento: NextPage = () => {
+  const { t } = useTranslation(["common", "perfezionamento"]);
   const [currentTab, setCurrentTab] = useState({ index: 0, visible: true });
   const transitionDuration = 500;
   const containerRef = useRef(null);
-  const tabsData = getCommonTabsData("tabs notification viewed 1");
-  const headingTitleData = getCommonHeadingTitleData(
-    "heading title notification viewed 1"
-  );
+
+  const heading: IHeadingTitleProps = {
+    title: t("heading.title", { ns: "perfezionamento" }),
+    subtitle: (
+      <>
+        {t("heading.subtitle_1", { ns: "perfezionamento" })}
+        <Link href="https://notifichedigitali.pagopa.it/faq#perfezionamento-cosa-significa">
```

Comment:
use relative link

Reply:
Using a relative path nextjs formats the path to '/lang/faq/#perfezionamento-cosa-significa' (it adds a '/' right before the '#') and the user is redirected to the 'faq' page but not to the proper faq section.

Reply:
I made the required change since it doesn't work well only on the local environment but it does on the built one.